types/Cargo.toml

```diff
@@ -60,10 +63,12 @@ default = ["p2p"]
 p2p = ["dep:libp2p-identity", "dep:multiaddr", "dep:serde_repr"]
 test-utils = ["dep:ed25519-consensus", "dep:rand"]
 wasm-bindgen = ["celestia-tendermint/wasm-bindgen"]
+tonic = ["dep:pbjson-types", "celestia-proto/tonic"]
 
 [package.metadata.docs.rs]
 features = ["p2p", "test-utils"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [package.metadata.cargo-udeps.ignore]
 development = ["indoc"]
+normal = ["prost-types"] # unused if tonic flag is enabled
```

Comment:
afaik, feature flags can only enable optional dependencies, so this would require adding `prost` flag. This in conjunction with logic to have only one of prost and tonic enabled, and propagation seemed more convoluted than having the dependency ignored here, though I'm not wedded to that conclusion :p